### ISSUE TITLE
Accessibility is sub-optimal

### ISSUE THREAD

**Issue by TyIsI** [dat-ecosystem/website#39]
Hi!

While I love the site and the design, beyond missing screen reader options, the font makes it really hard to read on any of my screens.

<img width="944" height="930" alt="Image" src="https://github.com/user-attachments/assets/42ab02c5-925b-460c-a861-5cad1e02c628" />

<img width="953" height="936" alt="Image" src="https://github.com/user-attachments/assets/4aa95c49-4700-41dc-b25f-497de7bdf56a" />

<img width="958" height="935" alt="Image" src="https://github.com/user-attachments/assets/8615f006-1d9c-4009-8028-a65a55a93fb0" />

I love the quirky old school Apple design though!

**Comment by ninabreznik** [comment]
Thank you so much for reaching out. 

Do you have any suggestion on how we could improve this? We are currently applying for a new grant so hopefully if we receive the funding wecould allocate some part of the funds to improve the accesibility. That would be really nice. But in the meanwhile we can maybe reach out to the community if anyone would be willing to contribute their time and help out with this issue. We are working with some students and maybe I could share the issue with them, but if you have any ideas or recommendations, we would really appreaciate it.
